plugins/ua_config_standard.c

```diff
@@ -5,12 +5,19 @@
 #include "ua_log_stdout.h"
 #include "ua_network_tcp.h"
 #include "ua_accesscontrol_default.h"
+#include "ua_types_generated.h"
+#include "ua_securitypolicy_none.h"
+#include "ua_types.h"
 
-/*******************************/
-/* Default Connection Settings */
-/*******************************/
+#define ANONYMOUS_POLICY "open62541-anonymous-policy"
+#define USERNAME_POLICY "open62541-username-policy"
 
-const UA_EXPORT UA_ConnectionConfig UA_ConnectionConfig_standard = {
+ /*******************************/
+ /* Default Connection Settings */
+ /*******************************/
+
+const UA_EXPORT UA_ConnectionConfig UA_ConnectionConfig_standard =
```

Comment:
Now that we have UA_ServerConfig_standard_new, can we just fill the struct in a proper function?
Those static assignments are just horrible to read.